docs/design/storage-node-deployment-windows-home.md

```diff
@@ -0,0 +1,146 @@
+# Storage Node Automatic Updates and Installation for Windows Home
+
+## Overview
+
+As more Storage Node Operators join the network we must ensure these nodes have a mechanism to automatically keep their nodes software up to date.
+If a Storage Node Operator fails to keep their node up to date with the minimum version required by satellites they will no longer be selected for upload or download requests.
+
+Currently we are using Docker for automatic updates which means we require Docker to be installed.
+Docker is being used for many other things as well, so we need to cover these cases as well.
+
```

Comment:
The SN should not decide if he needs to update, this decision should be done from update server. Every node should send his version to the update server and the server send back update or not. Only with this we have the control over the update process, and we can updates in waves and depending on there network importance. Very important nodes should be updated last, if there were no fails.